locales/en.default.schema.json

```diff
@@ -1013,7 +1013,7 @@
         },
          "header__9": {
           "content": "Follow on Shop",
-          "info": "Display follow button for your storefront on the Shop app. [Learn more](https://help.shopify.com/manual/online-store/themes/customizing-themes/follow-on-shop)"
+          "info": "To allow shoppers to follow your store on the Shop app from your storefront, Shop Pay must be enabled. [Learn more](https://help.shopify.com/manual/online-store/themes/customizing-themes/follow-on-shop)"
```

Comment:
Is `shoppers` a term we use in our content UX? I can find a few references to `customers` in our translation files and our help docs also use the same term:

<img width="974" alt="image" src="https://user-images.githubusercontent.com/37168033/227078943-5033fc60-c166-49f9-8055-8ff3f64300de.png">

Reply:
I got this text from UX, so I am going to allow them to chime in.

@bjorkeson thoughts and feelings?

Reply:
@Michael-Webster21 What do you think about this?